README.md

```diff
@@ -0,0 +1,216 @@
+<a name="#readme-top"></a>
```

Comment:
- [ ] This is the frontend app, however, much of the readme instructions in this repo are for the Rails backend.  Kindly make it so the frontend repo has the specific instructions to setup and run the frontend app correctly. 👍